python/modules/Stop0l_trigger.py

```diff
@@ -107,7 +110,27 @@ def get_efficiency(self, trigger_name, kinematic):
 	#print " my_xrange is ", my_xrange
         if findbix == -1:
             return 0, 0, 0
-        return eff.GetY()[findbix], eff.GetY()[findbix] - eff.GetErrorYlow(findbix), eff.GetY()[findbix] + eff.GetErrorYhigh(findbix)
+	
+	eff_central = eff.GetY()[findbix]
+	eff_down = eff_central - eff.GetErrorYlow(findbix)
+	eff_up = eff_central + eff.GetErrorYhigh(findbix)
+
+	if(trigger_name == "MET_low_dm_QCD"): 		#assign additional sys unc for MET trigger eff in low dm QCD CR
+		eff_METSig = self.effs["MET_low_dm_QCD_METSig"]
+        	my_xrange_METSig = np.array([eff_METSig.GetX()[i] - eff_METSig.GetErrorXlow(i) for i in range(eff_METSig.GetN()) ])
+        	my_xrange_METSig = np.append(my_xrange_METSig, 99999)
+        	findbix_METSig = np.searchsorted(my_xrange_METSig, kinematic)-1
```

Comment:
Why can't you use the same findbix as above? They have to be the same no?

Reply:
The findbix is different in the TEfficiency with METSig cut because TEfficiency ignores the first few bins with numerator=denominator=0